src/documents/WebGamesAndSimulations.md

```diff
@@ -0,0 +1,164 @@
+
+* Provide alternatives to charts. Give text descriptions and interpretations of the data, give a table of data.
+* A good description identifies:
+    * What is being shown (i.e. the purpose of the chart)
+    * Describe the values (relative sizes and differences between values).
+    * Describe any patterns to may help us understand the data.
+    * Summarize the trend / data pattern.
+* Also see:
+    * PSU’s chart accessibility guide - <a href="http://accessibility.psu.edu/images/charts/" rel="nofollow" target="_blank" class="link-external">http://accessibility.psu.edu/images/charts/</a>
+    * W3C's guide on describing complex images - <a href="http://www.w3.org/WAI/tutorials/images/complex/" rel="nofollow" target="_blank" class="link-external">http://www.w3.org/WAI/tutorials/images/complex/</a>
+
+If using a table of data:
+
+* Follow best practices for table formatting (i.e. proper use of table headers, no table nesting). <a href="https://www.webaccessibility.com/best_practices.php?technology_platform_id=8" rel="nofollow" target="_blank" class="link-external">See Data Table Best Practices at WebAccessibility.com</a>
+* Any math, symbols, units, or acronyms defined. For additional information about making math accessible, see [Mathematics](Mathematics.html).
+* Tables can be cumbersome to navigate (like with screen readers, small mobile displays, or screen magnifiers). Simplify and separate data tables if possible.
+
+## Poet Image Description Tool for Graphs
+
+The Poet Image Description Tool is a web-based tool for creating and providing guidelines to writing image descriptions for images in existing DAISY and EPUB books. This tool has resources and tools to help content authors create good image descriptions following an extended NCAM guideline. There is also a math editor which transcribes math into readable format,
+
+See: Poet Image Description Tool at <a href="http://diagramcenter.org/development/poet.html" rel="nofollow" target="_blank" class="link-external">http://diagramcenter.org/development/poet.html</a>.
+
+##  Cognitive
+
+Information should be presented in a clear concise manner, and where possible starting with general statements first with more detailed information following.
+
+* Instructions should be clear and concise, using a language that is appropriate.
+* If there is any possibility of confusion, make help readily available.
+* Avoid really dense content and interactions. Leverage semantic markup such as `<section>` elements with proper `<h1>` headings to help group content into navigable chunks.
+* Consider presenting information as needed, rather than presenting it all up front.
+* For example: Instead of printing instructions up front in plain text, instructions on how to use a control may be better presented when the user is hovering or focused on that control (like a tooltip). This helps reduce cognitive load and gives information only when it's needed.
```

Comment:
This example seems to be related to the previous bullet point. Perhaps it should simply be a continuation of that same bullet point? or a sub-bullet?